### PR TITLE
chore: Add workflow to keep them in sync

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Similar to https://github.com/cloudquery/cloudquery/blob/main/.github/workflows/endpoints.yml.
This PR creates a workflow to periodically update https://github.com/cloudquery/cloudquery/blob/796e82a30436e0df3a0d00fa82daad98cec015fb/plugins/source/gcp/client/services.go

### CODE REVIEW

File: plugins/source/gcp/client/services.go [hunk 1]
```diff
@@ -149,7 +149,6 @@ var GcpServices = map[string]bool{
 	"iam.googleapis.com":                           true,
 	"iamcredentials.googleapis.com":                true,
 	"iap.googleapis.com":                           true,
-	"ideahub.googleapis.com":                       true,
```
Comment (erezrokah) [inline plugins/source/gcp/client/services.go:152]: We don't use this so it's not breaking

File: .github/workflows/gcp-services.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: Generate GCP Services
+on:
+  schedule:
+    - cron: "60 * * * *"
```
Comment (yevgenypats) [inline .github/workflows/gcp-services.yml:4]: is it every hours? can we do weekly basis?

Comment (erezrokah) [reply .github/workflows/gcp-services.yml:4]: This is actually wrong syntax 🤦 
I'll update it to run every Monday morning

**Comment by erezrokah** [comment]
I'll go ahead and merge this as it's blocking another PR

**Pull request merged**